inspire_matcher/validators.py

```diff
@@ -107,11 +107,6 @@ def persistent_identifier_validator(record, result):
 
 
 def arxiv_eprints_validator(record, result):
-    record_eprints = get_value(record, "arxiv_eprints.value", [])
-    result_eprints = get_value(result, "_source.arxiv_eprints.value", [])
-
-    for record_eprint in record_eprints:
-        if record_eprint in result_eprints:
-            return False
-
-    return True
+    record_eprints = set(get_value(record, "arxiv_eprints.value", []))
+    result_eprints = set(get_value(result, "_source.arxiv_eprints.value", []))
+    return bool(record_eprints & result_eprints)
```

Comment:
no, there are two valid cases where we should return true:
* this one, where there's an overlap (corner case)
* the regular case, where any of the sets is empty

Reply:
@michamos updated.